docs/tutorials.rst

```diff
@@ -24,6 +24,7 @@ Notebooks
 
    notebooks/first_steps.ipynb
    notebooks/intro_maps.ipynb
+   notebooks/modeling.ipynb
```

Comment:
If the idea of the notebook is similar to the maps intro, then maybe rename it to `intro_models.ipynb`? I think "modeling" maybe suggests, that there is fitting involved...